media/starboard/starboard_renderer.cc

```diff
@@ -144,25 +144,43 @@ void StarboardRenderer::Initialize(MediaResource* media_resource,
     video_stream_->EnableBitstreamConverter();
   }
 
-#if COBALT_MEDIA_ENABLE_ENCRYPTED_PLAYBACKS
-  base::AutoLock auto_lock(lock_);
+  // TODO(b/378958007) ensure SetCdm has been called before
```

Comment:
TODO(cobalt, b/378958007)

We are trying to tag the TODO comments with "cobalt" for easy analysis later on.

Reply:
done